dev/io.openliberty.io.grpc.1.0/bnd.bnd

```diff
@@ -16,7 +16,7 @@
 -sub: *.bnd
```

Comment:
Update copyright date range to include 2023

Reply:
Done